### PR TITLE
Update to allow *int64 for Datetime64 type

### PR DESCRIPTION
**Pull request by rtkaratekid**

### CODE REVIEW

**Comment by rtkaratekid** [comment]
If other changes need to be made to allow pointers for other types, please feel free to make them. I've only tested this one though.

File: lib/column/datetime64.go [hunk 1]
```diff
@@ -141,6 +141,8 @@ func (dt *DateTime64) AppendRow(v interface{}) error {
 	switch v := v.(type) {
 	case int64:
 		datetime = v
+	case *int64:
+		datetime = *v
```
Comment (kshvakov) [inline lib/column/datetime64.go:145]: The value can be nil. It's potentially `invalid memory address or nil pointer dereference`

**Comment by rtkaratekid** [comment]
any update on this?

**Comment by kshvakov** [comment]
Also, you should add `*int64` to the Append method

**Comment by rtkaratekid** [comment]
Anything else I need to do to get this approved?

**Comment by rtkaratekid** [comment]
Just checking in to see if I need to do anything else.

**Comment by gingerwizard** [comment]
I've tested this and im satisfied it works. I will follow up with a test enhancement to cover this. In general we will be lenient on types moving forward but not allow any conversions which result in a loss of precision.

Thanks @rtkaratekid for this. Apologies it took soo long. Change of maintainers.

**Comment by rtkaratekid** [comment]
@gingerwizard No problem, I'm just glad it's accepted.

**Pull request merged**